transport/http/outbound.go

```diff
@@ -53,7 +53,18 @@ var (
 	_ introspection.IntrospectableOutbound = (*Outbound)(nil)
 )
 
-var defaultURLTemplate, _ = url.Parse("http://localhost")
+const http2AuthorityPseudoHeader = ":authority"
+
+var (
+	defaultURLTemplate, _ = url.Parse("http://localhost")
+	// from https://tools.ietf.org/html/rfc7540#section-8.1.2.3
+	http2PseudoHeaders = []string{
```

Comment:
is this exhaustive? or rather, any psudo header started with `:` is un-parsable/invalid in HTTP/1 right?

Reply:
This is exhaustive from the RFC. My intention here is strip those http2 specific ones and have a followup PR that fails fast on other invalid ones starting with `:`